Rakefile

```diff
@@ -1,9 +1,28 @@
 require 'rake'
 require 'rake/testtask'
 
-desc 'rake with no argument will run "rake test"'
-task default: :test
+require_relative 'lib/tasks/exercise'
+require_relative 'lib/tasks/exercise_test_tasks'
+require_relative 'lib/tasks/rake_arguments'
```

Comment:
I still only want to include `exercise_test_tasks` here.
The other things *are* references here, but *only* because `exercise_test_tasks` is.